content/en/docs/platforms/web/basics.md

```diff
@@ -113,7 +113,11 @@ static_resources:
   - name: echo_service
     connect_timeout: 0.25s
     type: LOGICAL_DNS
-    http2_protocol_options: {}
+    typed_extension_protocol_options:
+      envoy.extensions.upstreams.http.v3.HttpProtocolOptions:
+        "@type": type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions
```

Comment:
Do you know what is the specific Envoy version where this starts to work?

Reply:
In 1.23 it works for sure (didn't test in earlier versions). And current supported versions are 1.25+

Reply:
I will update also in the grpc-web repo as well (I only searched for grpc.io for occurrences). Thanks for pointing me to it.

Reply:
> In 1.23 it works for sure (didn't test in earlier versions). And current supported versions are 1.25+

Thanks for the info!!

> I will update in the grpc-web repo as well (I only searched for grpc.io for occurrences). Thanks for pointing me to it.

@lcostea Oh that's great! Thank you so much!! 😊